MN/Algoritmi/Metode-Iterative/Ecuatii-Neliniare/m_bisectiei.m

```diff
@@ -0,0 +1,37 @@
+function [m] = m_bisectiei(f, tol, a, b)
```

Comment:
```matlab
function [x i] = BisectionMethod(f, a, b, tol, max_iter)
```
- f e un function handler dat ca parametru (o functie, nu un polinom) --- de exemplu, poti apela BisectionMethod(@sin, a, b, tol, max_iter) sau BisectionMethod(@functiaMeaF, a, b, tol, max_iter)
- a, b, tol sunt ce erau in laborator
- max_iter --- nu vrem mai mult de max_iter pasi; daca nu converge, ghinion
- returnam x (radacina) si i (pasul la care am ajuns in iteratie)